### PR TITLE
Update default.json

### PR DESCRIPTION
**Pull request by connorpgpmcelroy**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-7253
https://tools.hmcts.net/jira/browse/EUI-7269
https://tools.hmcts.net/jira/browse/EUI-7322
https://tools.hmcts.net/jira/browse/EUI-7013

### Change description ###
Adding other relevant services to waSupportedJurisdictions
EUI-7013 is about fixing the failing tests for Hearings

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by Kartheek-S — APPROVED** [review]

File: config/default.json [hunk 1]
```diff
@@ -97,7 +97,7 @@
   "microservice": "xui_webapp",
   "sessionSecret": "secretSauce",
   "globalSearchServices": "IA,CIVIL,PRIVATELAW",
-  "waSupportedJurisdictions": ["IA"],
+  "waSupportedJurisdictions": "IA,CIVIL,PRIVATELAW",
```
Comment (udaydenduluri33) [inline config/default.json:100]: This should only be
`IA`
We will add the other services using Flux.

**Review by udaydenduluri33 — APPROVED** [review]

**Review by Kartheek-S — APPROVED** [review]

**Review by Mo-Lala84 — DISMISSED** [review]
approved

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by udaydenduluri33 — APPROVED** [review]

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by udaydenduluri33 — APPROVED** [review]

**Review by tarikdogan-hmcts — APPROVED** [review]

**Pull request merged**